src/screens/Welcome.tsx

```diff
@@ -2,31 +2,48 @@ import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import { StackScreenProps } from '@react-navigation/stack';
 import { Button } from 'react-native-elements';
+import { signOutUser } from '../firebase/auth';
 
 const WelcomeScreen: React.FC<StackScreenProps<any>> = ({ navigation }) => {
   return (
     <View style={styles.container}>
       <Text>Welcome screen!</Text>
 
       <View style={styles.buttons}>
-        <Button title="Sign in" buttonStyle={styles.button} onPress={() => navigation.navigate('Sign In')} />
-        <Button title="Sign up" type="outline" buttonStyle={styles.button} onPress={() => navigation.navigate('Sign Up')} />
+        <Button
+          title="Sign in"
+          buttonStyle={styles.button}
+          onPress={() => navigation.navigate('Sign In')}
+        />
+        <Button
+          title="Sign up"
+          type="outline"
+          buttonStyle={styles.button}
+          onPress={() => navigation.navigate('Sign Up')}
+        />
+
+        <Button
+          title="temp sign out"
+          type="outline"
+          buttonStyle={styles.button}
+          onPress={() => signOutUser()}
+        />
       </View>
     </View>
   );
-}
+};
 
 const styles = StyleSheet.create({
```

Comment:
this should be added to a separate styles file + this welcome screen should be nested within a directory (probably Authentication)